kartothek/api/discover.py

```diff
@@ -117,8 +118,7 @@ def discover_datasets_unchecked(uuid_prefix, store, filter_ktk_cube_dataset_ids=
         }
 
     result = {}
-    # sorted iteration for determistic error messages in case
-    # DatasetMetadata.load_from_store fails
+    # sorted iteration for determistic error messages in case DatasetMetadata.load_from_store fails
```

Comment:
why are the comments weird again?

Reply:
The comments were in straight line in klee2 code.  But was tweaked while transforming to ktk. So reverted them back to  old comments.